theano/sandbox/cuda/opt.py

```diff
@@ -1587,7 +1580,7 @@ def local_conv_gemm(node):
             # need to flip the kernel for valid convolution
             kern = kern[:, :, ::-1, ::-1]
             # By default use GpuCorrMM
-            rval = GpuCorrMM(border_mode, subsample)(
+            rval = GpuCorrMM(**node.op._props_dict())(
```

Comment:
Node is GpuConv and it don't implement \__props__ so this would fail if used.

GpuConv is hard to implement this \__props__, and is obsolete. So just don't change this line.